sros2/sros2/api/__init__.py

```diff
@@ -116,6 +122,26 @@ def check_openssl_version(openssl_executable):
         raise RuntimeError('need openssl 1.0.2 minimum')
 
 
+def write_key(
```

Comment:
This is looking good, I just have one last question: this API is public, no? Are we sure we want to make functions like this public as well? Or should we prepend them with an underscore? If so, we should do that now, otherwise changing it in the future is technically breaking API.

Reply:
made private